docs/reference/controllers.md

```diff
@@ -59,6 +59,25 @@ For example, the `<div>` and `<h1>` below are part of the controller's scope, bu
 </main>
 ```
 
+## Nested Scopes
+
+When controllers are nested, they're aware of _only_ their own scope and not the scope of any controllers nested within them.
```

Comment:
I'd change to `they are`.   But perhaps `they` should be explicit about the parent.   So `When nested, the parent controller is only aware of their own scope...`

Reply:
Ya, the wording was a bit tricky. I guess technically every controller is only aware of its own scope, no matter where it is in the tree.

What do you think about something like the following? Think it's necessary to be explicit about the `parent`?

```
When nested, each controller is only aware of its own scope excluding the scope of any controllers nested within.
```

Reply:
This just burned me and is important to document sooner rather than later as the current documentation is incorrect:  "When Stimulus connects a controller to an element, that element and all of its children make up the controller's scope."

While the current behavior is understandable and probably even desirable in most cases, there are cases where it's not.  Wondering if there is anyway to have an option to allow a controllers scope to include the Dom of nested controllers?